.github/workflows/docker-build-static.yml

```diff
@@ -149,25 +152,19 @@ jobs:
   build-aarch64:
     runs-on: ubuntu-latest
     container:
-      image: rust:1.61
+      #image: rust:1.62
+      image: nitnelave/rust-dev:latest
       env:
-        CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: aarch64-linux-gnu-gcc
+        CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_LINKER: aarch64-linux-musl-gcc
         CARGO_TERM_COLOR: always
         RUSTFLAGS: -Ctarget-feature=+crt-static
         CARGO_HOME: ${GITHUB_WORKSPACE}/.cargo
     steps:
-      - name: add arm64 architecture
-        run: dpkg --add-architecture arm64
-      - name: install runtime
-        run: apt update && apt install -y gcc-aarch64-linux-gnu g++-aarch64-linux-gnu libc6-arm64-cross libc6-dev-arm64-cross tar
-      - name: smoke test
-        run: rustc --version
       - name: Checkout repository
         uses: actions/checkout@v2
-      - name: add arm64 target
-        run: rustup target add aarch64-unknown-linux-gnu
       - name: smoke test
-        run: rustc --version
+        run: |
```

Comment:
Why the pipe?

Reply:
used to be test something but removed at the end.